drivers/overlay/ostweaks_linux.go

```diff
@@ -1,17 +1,41 @@
 package overlay
 
 import (
+	"fmt"
 	"io/ioutil"
 	"path"
+	"strconv"
 	"strings"
 
 	"github.com/sirupsen/logrus"
 )
 
-var sysctlConf = map[string]string{
-	"net.ipv4.neigh.default.gc_thresh1": "8192",
-	"net.ipv4.neigh.default.gc_thresh2": "49152",
-	"net.ipv4.neigh.default.gc_thresh3": "65536",
+type conditionalCheck func(val1, val2 string) bool
+
+type osValue struct {
+	value   string
+	checkFn conditionalCheck
+}
+
+var osConfig = map[string]osValue{
+	"net.ipv4.neigh.default.gc_thresh1": {"8192", checkHigher},
+	"net.ipv4.neigh.default.gc_thresh2": {"49152", checkHigher},
+	"net.ipv4.neigh.default.gc_thresh3": {"65536", checkHigher},
+}
+
+func newValIsHigher(val1, val2 string, check conditionalCheck) bool {
+	if check == nil || check(val1, val2) {
+		fmt.Println("Yep doing something")
+		return true
+	}
+	fmt.Println("Nope")
```

Comment:
remove this

Reply:
Whoops, this print and the previous one shouldn't have slipped through. Cleaned it up during one of my edits. I'll get those fixed.